### PR TITLE
Docs updates

### PR DESCRIPTION
**Pull request by mtargonski**
I was going over boneIO videos/tutorials and documentations, and figured out I could at the same time suggest some minor tweaks for the documentation, nothing big really.

### CODE REVIEW

File: docs/02-configuration/cover.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ sidebar_position: 11
 
 ![Cover](/img/cover.png)
 
-Cover module. It can be used with cover board and relay board as well.
+Cover module. It can be used with cover board and relay board as well. Used for roller shutters.
```
Comment (pszafer) [inline docs/02-configuration/cover.md:9]: We decided to use only `cover` name, same as esphome, tasmota and Home Assistant.

File: docs/02-configuration/modbus-sensors.md [hunk 2]
```diff
@@ -39,9 +39,9 @@ modbus_sensors:
...
+- **address** (**Required**, string) - modbus address of the device you want to communicate with.
 - **model** (**Required**, string) - model schema to use. Check table above to get model name.
 - **update_interval** (_Optional_, timeperiod, default: 60 seconds) - update interval. Modbus connection is pretty cpu intensive so customize with care. Minimum is 1 second, but recommended minimum is 10 secs.
 
-[^1]: Those energy metters can be found in two versions and only one support Modbus - sometimes it's marked with suffix `M` (`SDM120M` / `SDM6301M`), other time `Modbus` is mentioned in the title. Don't miss it with cheaper version with suffix `D` (`SDM120D` / `SDM6301D`) - it doesn't support Modbus communication so it cannon't be connected with Bone.io.
+[^1]: Those energy metters can be found in two versions and only one supports Modbus - sometimes it's marked with suffix `M` (`SDM120M` / `SDM6301M`), other time `Modbus` is mentioned in the title. Don't confuse it with cheaper version with suffix `D` (`SDM120D` / `SDM6301D`) - it doesn't support Modbus communication so it cannon't be connected with Bone.io.
```
Comment (pszafer) [inline docs/02-configuration/modbus-sensors.md:47]: one typo missed by you `cannon't`

File: docs/02-configuration/cover.md (hunk 1 above)
Comment (mtargonski) [reply docs/02-configuration/cover.md]: removed added text

File: docs/02-configuration/modbus-sensors.md (hunk 2 above)
Comment (mtargonski) [reply docs/02-configuration/modbus-sensors.md]: yep. fixed

**Pull request merged**